src/presentation/components/TopBar/TopBar.module.scss

```diff
@@ -5,16 +5,15 @@
 
   &__item {
     &TextCollection {
+      display: flex;
+      justify-content: space-evenly;
       padding-top: 2rem;
-      margin-left: 1rem;
-    }
-
-    &TextFAQ {
-      padding-top: 2rem;
-      margin-left: -10rem;
     }
 
     &Button {
+      display: flex;
+      justify-content: end;
+      align-items: center;
```

Comment:
Its ok or you meant : `justify-content: flex-end;` ?

Reply:
oh yes, you are right. justify-content: flex-end; would be the correct solution